src/application.coffee

```diff
@@ -77,7 +77,7 @@ logTypes =
 		humanName: 'Restarting application'
 
 logSystemMessage = (message, obj, eventName) ->
-	logger.log({ message, isSystem: true })
+	logger.log({ m: message, s: true })
```

Comment:
It won't, you can pass any true-ish thing.

On Thu, Jul 21, 2016 at 9:02 PM, Page- <notifications@github.com> wrote:

> In src/application.coffee
> <https://github.com/resin-io/resin-supervisor/pull/213#discussion_r71755731>
> :
>
> > @@ -77,7 +77,7 @@ logTypes =
> >  		humanName: 'Restarting application'
> >
> >  logSystemMessage = (message, obj, eventName) ->
> > -	logger.log({ message, isSystem: true })
> > +	logger.log({ m: message, s: true })
>
> just noticed this, it'd be nice to use s: 1 to save 3 bytes per message
> easily - I'm not sure the ui would even need any changes, @dfunckt
> <https://github.com/dfunckt> @emirotin <https://github.com/emirotin> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/resin-io/resin-supervisor/pull/213/files/c5ecbfe0d3ea6026f6bfc903b48548eb814a3047#r71755731>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgGCBfrZ-AHHcwgtsVKIeABqeYK3PAPks5qX7Q5gaJpZM4JO8gF>
> .
>



-- 
Eugene Mirotin
Senior Frontend Engineer
site: Resin.io <https://resin.io/>, twitter: @resin_io
<https://twitter.com/resin_io>